internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -116,7 +116,8 @@ spec:
 
         /usr/bin/kinit -V $(cat /mnt/service-account-secret/principal) -k -t /tmp/keytab
 
-        # check if this fbc fragment is opt-in
+        set -x
+        # chek if this fbc fragment is opt-in
```

Comment:
unintended change with typo?

Reply:
not even noticed. Thanks for mentioning!